src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/changeset/RelationChangeSetHandler.java

```diff
@@ -54,7 +55,7 @@ public RelationChangeSetHandler(final Atlas atlas, final RelationChangeSet chang
     public Atlas applyChanges()
     {
         // Log original Atlas statistics
-        logger.info(atlasStatistics(super.getAtlas()));
+        logger.info("Before Slicing Relations: " + atlasStatistics(super.getAtlas()));
```

Comment:
If those logger calls generate a lot of processing, it might be nice to wrap them in `if (logger.isInfoEnabled())` so it does not trigger when the logging level is at WARN or higher?

Reply:
Only a single line. Here's an example. Before:
2018-01-05 11:52:12 INFO  [main] RelationChangeSetHandler:58 - Points: 648709 Lines: 57643 Relations: 36
2018-01-05 11:52:16 INFO  [main] RelationChangeSetHandler:77 - Points: 648709 Lines: 57643 Relations: 44

After:
2018-01-05 11:52:12 INFO  [main] RelationChangeSetHandler:58 - Before Slicing Relations: Points: 648709 Lines: 57643 Relations: 36
2018-01-05 11:52:16 INFO  [main] RelationChangeSetHandler:77 - After Slicing Relations: Points: 648709 Lines: 57643 Relations: 44

Reply:
Wanted to add something to different the two.

Reply:
I meant more if `atlasStatistics(super.getAtlas())` takes some time to process, only process it when the logger will actually print it!

Reply:
Good point, will add this.